schemas/v3.1/schema.yaml

```diff
@@ -0,0 +1,809 @@
+        properties:
+          implicit:
+            allOf:
+              - required:
+                  - scopes
+              - $ref: '#CommonFlow'
+              - $ref: '#AuthorizationFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          password:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          clientCredentials:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          authorizationCode:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#AuthorizationFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+        unevaluatedProperties: false
+        $defs:
+          Common:
+            $anchor: CommonFlow
+            properties:
+              refreshUrl:
+                type: string
+                format: uri-reference
+              scopes:
+                type: object
+                additionalProperties:
+                  type: string
+          Token:
+            $anchor: TokenFlow
+            required:
+              - tokenUrl
+            properties:
+              tokenUrl:
+                type: string
+                format: uri-reference
+          Authorization:
+            $anchor: AuthorizationFlow
+            required:
+              - authorizationUrl
+            properties:
+              authorizationUrl:
+                type: string
+                format: uri-reference
+      Link:
+        $anchor: Link
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            oneOf:
+              - properties:
+                  operationRef:
+                    type: string
+                    format: uri-reference
+                  operationId: false
+              - properties:
+                  operationRef: false
+                  operationId:
+                    type: string
+            properties:
+              parameters:
+                type: object
+                additionalProperties: true
+              requestBody: true
+              server:
+                $ref: '#Server'
+            unevaluatedProperties: false
+      Callback:
+        $anchor: Callback
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+      Schema:
+        $anchor: Schema
+        oneOf:
+          - $ref: '#Reference'
```

Comment:
```suggestion
```

This reference is redundant, as a draft2020-12 schema will of course accept the $ref keyword itself.

Reply:
(this is no longer redundant if the '#Reference' definition is extended to include 'summary' and 'description' properties and "unevaluated: false".)